packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -191,16 +227,17 @@ export interface ISegment extends IMergeNodeCommon, Partial<IRemovalInfo> {
     /**
      * Acks the current segment against the segment group, op, and merge tree.
      *
-     * Throws error if the segment state doesn't match segment group or op.
-     * E.g. Segment group not first is pending queue.
-     * Inserted segment does not have unassigned sequence number.
-     *
-     * Returns true if the op  modifies the segment, otherwise false.
+     * @param segmentGroup - Pending segment group associated with this op.
+     * @param opArgs - Information about the op that was acked
+     * @param attribution - If specified, any attribution data associated with this segment.
+     * @returns - true if the op modifies the segment, otherwise false.
      * The only current false case is overlapping remove, where a segment is removed
      * by a previously sequenced operation before the current operation is acked.
-     *
+     * @throws - error if the segment state doesn't match segment group or op.
+     * E.g. if the segment group is not first in the pending queue, or
+     * an inserted segment does not have unassigned sequence number.
      */
-    ack(segmentGroup: SegmentGroup, opArgs: IMergeTreeDeltaOpArgs): boolean;
+    ack(segmentGroup: SegmentGroup, opArgs: IMergeTreeDeltaOpArgs, attribution?: unknown): boolean;
```

Comment:
unknown seems like the wrong type here, this will always be seq if it exists, right? so `number`

Reply:
do we even need this change. it doesn't like the method implementation uses it. this is why i generally prefer non-optional, but undefined allowed parameters, as you get better compiler help

Reply:
yeah, my bad--failed to remove. fair point about required `undefined`.